### PR TITLE
Fixes date issue in snapshot test

### LINKED ISSUE #338: Some code in the measure tables tests are time-dependent, causing jest tests to fail
The just-landed #336 is failing CI:

https://circleci.com/gh/mozilla/missioncontrol/992?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

It appears that some parts of the measure table code are time-dependent. We should something like the strategy here to make sure a consistent result is returned by the react component:

https://jestjs.io/docs/en/snapshot-testing#2-tests-should-be-deterministic

@kagehina919 could you take a look at doing this? I think it should be pretty easy.

### PR DESCRIPTION
**Pull request by kagehina919**
Fixes #338. 

`lastUpdated` has been updated by a static function `Date.now = jest.fn(() => 1482363367071);`.
@wlach please review.

### CODE REVIEW

File: frontend/test/components/measuretable.spec.jsx [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { shallow, configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import MeasureTable from '../../ui/measuretable';
 
+Date.now = jest.fn(() => 1482363367071);
```
Comment (wlach) [inline frontend/test/components/measuretable.spec.jsx]: Let's make this a little more human readable:

```js
const referenceDate = new Date("2018-11-20T15:00:00Z");
Date.now = jest.fn(() => referenceDate.getTime());
```

I think then you can revert the changes below to change lastUpdated to use referenceDate instead of Date.now(), and the snapshots should be more realistic (instead of swapping a reference date for an integer). Or at least I think so. Give it a try. :smile:

Comment (kagehina919) [reply frontend/test/components/measuretable.spec.jsx]: > and the snapshots should be more realistic (instead of swapping a reference date for an integer)

I have updated the reference date. I am sorry I don't get what you mean by this? :sweat_smile:

File: frontend/test/components/measuretable.spec.jsx [hunk 2]
```diff
@@ -66,7 +70,7 @@ describe('MeasureTable', () => {
             adjustedRate: 1.19,
           },
         ],
-        lastUpdated: '2018-11-20T15:00:00Z',
+        lastUpdated: Date.now(),
```
Comment (wlach) [inline frontend/test/components/measuretable.spec.jsx]: You should be able to just pass referenceDate in here

File: frontend/test/components/measuretable.spec.jsx [hunk 3]
```diff
@@ -80,6 +84,8 @@ describe('MeasureTable', () => {
       />
     );
 
-    expect(component).toMatchSnapshot();
+    expect(component).toMatchSnapshot({
+      lastUpdated: Date.now(),
+    });
```
Comment (wlach) [inline frontend/test/components/measuretable.spec.jsx]: You can remove the arguments here, they should not be necessary (after you update the snapshot)

File: frontend/test/components/measuretable.spec.jsx (hunk 1 above)
Comment (wlach) [reply frontend/test/components/measuretable.spec.jsx]: Oh, I just mean it will more realistically reflect the data that is passed into measureTable in the real world. There we pass a date string, not an integer representation of unix time.
